[docs] Document TextField helperText height (#22146)
mui-org/material-ui: docs/src/pages/components/text-fields/text-fields.md

In the future, we might provide a [number input component](https://github.com/mui-org/material-ui/issues/19154).

### Helper text

> The helper text prop affects the height of the text field. If two text fields, are placed side by side, one with a helper text and one without, they will have different heights. For example:

Reviewer: ```suggestion
The helper text prop affects the height of the text field. If two text fields are placed side by side, one with a helper text and one without, they will have different heights. For example:
```

In the future, we might provide a [number input component](https://github.com/mui-org/material-ui/issues/19154).

### Helper text

> The helper text prop effects the height of the text field. If two text fields, one with a helper text one without, are placed side by side they will be at different heights. For example the following code: 

Reviewer: ```suggestion
The helper text prop affects the height of the text field. If two text fields, are placed side by side, one with a helper text and one without, they will have different heights. For example the following code: 
```

```jsx
<div style={{ display: 'flex', alignItems: 'center' }}>
	<TextField helperText="Please enter your name" />
	<TextField />
</div>
```

Will result in this:

> ![helper-text-1](/static/images/text-fields/helper-text-1.png)

Reviewer: Should we have a live example here?

Author: Ok sure I can change the images to live versions

> This can be fixed by passing a space character as a prop to text fields that do not have helper text:

Reviewer: ```suggestion
This can be fixed by passing a space character to the `helperText` prop:
```
